sdk/src/main/scala/com/horizen/account/block/AccountBlockHeader.scala

```diff
@@ -35,7 +36,7 @@ case class AccountBlockHeader(
                                stateRoot: Array[Byte],
                                receiptsRoot: Array[Byte],
                                forgerAddress: AddressProposition,
-                               baseFee: Long,
+                               baseFee: BigInteger,
                                gasUsed: Long,
                                gasLimit: Long,
```

Comment:
We need to verify the value of `gasLimit`.
At the moment malicious forger may setup any value and we will accept it.
Let's update `semanticValidity` method below and check:
0) baseFee > 0
1) gasLimit == FeeUtils.GAS_LIMIT
2) gasUsed >= 0 && gasUsed  <= gasUsed